aixplain/modules/agent/__init__.py

```diff
@@ -265,15 +280,29 @@ def run_async(
             resp = r.json()
             logging.info(f"Result of request for {name} - {r.status_code} - {resp}")
 
-            poll_url = resp["data"]
-            response = {"status": "IN_PROGRESS", "url": poll_url}
+            poll_url = resp.get("data")
+            execution_stats = resp.get("executionStats")
+            used_credits = resp.get("usedCredits", 0.0)
+            run_time = resp.get("runTime", 0.0)
+
+            return AgentResponse(
+                status=ResponseStatus.IN_PROGRESS,
+                url=poll_url,
+                input=input_data,
+                session_id=session_id or "",
+                execution_stats=execution_stats,
```

Comment:
Remove session_id and execution_stats. These fields do not exist during progress.

Reply:
Should I keep them for Success? Or should I completely remove them?